lib/domain/goal.js

```diff
@@ -0,0 +1,24 @@
+export default class Goal {
+  constructor({ agreedDate, targetReviewDate, text, useAsPhp }) {
+    this.agreedDate = agreedDate ? agreedDate : this.todaysDate();
```

Comment:
is it easier to write it as:
this.agreedDate = agreedDate || this.todaysDate();
?